portal/templates/utilisation-report-service/utilisation-report-upload/check-the-report.njk

```diff
@@ -18,7 +18,7 @@
     </h1>
     {% if validationErrors %}
         <div class="govuk-form-group govuk-form-group--error govuk-!-margin-top-7">
-            <h2 class="govuk-heading-m" id="validation-errors-table">Errors found in {{ filename }}</h2>
+            <h2 class="govuk-heading-m" id="validation-errors-table" tabIndex="-1">Errors found in {{ filename }}</h2>
```

Comment:
Ooh is it `tabIndex` or `tabindex`?
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex suggests lowercase `i` but I don't know if that's important?

Reply:
Ah whoops I'm still living in a react world 😅